src/astro/sql/operators/sql_dataframe.py

```diff
@@ -1,35 +1,68 @@
 import inspect
```

Comment:
Not strictly needed for this PR but Should we rename the module from `sql_dataframe` to just `dataframe` ?

Reply:
Worth discussing after Airflow summit. Won't affect user facing interfaces.

Reply:
@kaxil I added as a criteria for: https://github.com/astronomer/astro-sdk/issues/348, so we don't forget